README.md

```diff
@@ -0,0 +1,125 @@
+# BoutrosLab.plotting.general
+
+1. [Description](#description)
+2. [Included plots](#included-plots)
+3. [Installation](#installation)
+4. [Function reference](#function-reference)
+5. [Resources](#resources)
+6. [Getting help](#getting-help)
+7. [Contributors](#contributors)
+8. [Citation information](#citation-information)
+9. [License](#license)
+
+## Description
+
+Contains several plotting functions such as barplots, scatterplots, heatmaps, as well as functions to combine plots and assist in the creation of these plots. 
+These functions will give users great ease of use and customization options in broad use for biomedical applications, as well as general purpose plotting. 
+Each of the functions also provides valid default settings to make plotting data more efficient and producing high quality plots with standard colour schemes simpler. 
+All functions within this package are capable of producing plots that are of the quality to be presented in scientific publications and journals. 
+P'ng et al.; [BPG: Seamless, automated and interactive visualization of scientific data](doi:10.1186/s12859-019-2610-2); BMC Bioinformatics 2019.
+
+## Included plots
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12859-019-2610-2/MediaObjects/12859_2019_2610_Fig1_HTML.png?as=webp)
+
+Available chart-types. The basic chart-types available in BPG: **a** density plot, **b** boxplot, **c** violin plot, **d** segplot, **e** strip plot, **f** barplot, **g** scatterplot, **h** histogram, **i** qqplot fit, **j** qqplot comparison, **k** Manhattan plot, **l** polygon plot, **m** heatmap, **n** dotmap and **o** hexbinplot.
+
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs12859-019-2610-2/MediaObjects/12859_2019_2610_Fig2_HTML.png?as=webp)
+
+Multiplot example. BPG's create.multiplot function is able to combine multiple chart-types together into a single figure as shown above.
```

Comment:
We may want to replace `create.multiplot` with `create.multipanelplot` as `create.multipanelplot` is considered an improved version of `create.multiplot`. We may also want to add notes to suggest using `create.multipanelplot` instead of `create.multiplot`. @pboutros any thoughts on this?

Reply:
I agree in general, but if there's a really nice example that hasn't been recreated in `create.multipanelplot` (yet) fine to show the `create.multiplot` version in the meantime (and then create an issue for updating the code moving forward).  The general idea is that `create.multipanelplot` is preferred, but both are still supported for now.  In the medium-term,  `create.multiplot` will be deprecated